src/context/VideoContext.js

```diff
@@ -14,6 +17,7 @@ const VideoProvider = ({ children }) => {
   const [state, dispatch] = useReducer(videoReducer, initialState);
 
   useEffect(() => {
+    dispatch({ type: "LOADING" });
```

Comment:
I think this dispatch is for Video Reducer. But there is no  LOADING type declared in the reducer. Please correct me If I am missing something.

Reply:
it's declared. I missed dispatching the loading in the previous PR. so I'm just refactoring it here.